.travis.yml

```diff
@@ -1,13 +1,12 @@
 language: node_js
 sudo: false
 node_js:
-- '4'
-- '6'
-- '7'
-- '8'
-- '9'
+  - 4
```

Comment:
can you remove Node 4 as well? It won't be supported anymore in a month, so it's better dropping support from pino@5.